lab-ron/lib/greet.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function greet(name){
+  const blank = name || 'Ron Oh';
```

Comment:
This isn't exactly to specification, the assignment asks for a function which has a parameter name and returns 'hello ' + name, it should be exactly that. If you remove this line and change input to name on line 5 that will correct the issue.